docs/dev-guide/contributing/dev_setup.rst

```diff
@@ -206,6 +200,31 @@ follow.
     Fortunately, the code you are working on will be shared from your host via NFS so your work
     should have data safety.
 
+#. You can use SSHFS rather than NFS. The downside is SSHFS does not perform quite as well as NFS,
+   but the upside is you do not need to configure or run NFS, nor do you need to allow Vagrant to
+   edit your /etc/exports file. At the time of this writing, the ``vagrant-sshfs`` package is not
+   yet in Fedora, although the package is in the process of being reviewed. The author provides a
+   COPR repository you can enable to install the RPM::
```

Comment:
It did creep a little, but it is in a separate commit. I can back it out, make a new PR, and have someone review it, but... If I buy you a coffee when I'm back in Raleigh...

:coffee: